src/Platform.Handlers/src/Xamarin.Platform.Handlers/Handlers/Stepper/StepperHandler.iOS.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Drawing;
+using UIKit;
+
+namespace Xamarin.Platform.Handlers
+{
+	public partial class StepperHandler : AbstractViewHandler<IStepper, UIStepper>
+	{
+		protected override UIStepper CreateView()
+		{
+			var uIStepper = new UIStepper(RectangleF.Empty);
+			uIStepper.ValueChanged += OnValueChanged;
+			return uIStepper;
+		}
+
+		protected override void DisposeView(UIStepper uIStepper)
+		{
+			uIStepper.ValueChanged -= OnValueChanged;
+			base.DisposeView(uIStepper);
+		}
+
+		public static void MapMinimum(IViewHandler handler, IStepper slider) => (handler as StepperHandler)?.UpdateMinimum();
```

Comment:
we should use direct casting, and no null check. if this happen, we'd better fail (also, @PureWeen has another PR to gentrify this)

Reply:
also, you can actually set the value directly, without defining an UpdateIncrement method

Reply:
This has changed a bit since the comment.